opennlp-tools/src/main/java/opennlp/tools/ml/BeamSearch.java

```diff
@@ -113,9 +113,7 @@ public BeamSearch(int size, MaxentModel model, int cacheSize) {
         }
 
         double[] temp_scores = new double[scores.length];
-        for (int c = 0; c < scores.length; c++) {
-          temp_scores[c] = scores[c];
-        }
+        System.arraycopy(scores, 0, temp_scores, 0, scores.length);
```

Comment:
We did reject this change in the past, but now this seems to run just as fast as the for loop.

+1 to keep this